config/locales/en-US.yml

```diff
@@ -242,8 +242,9 @@
           html: Sent by <b>%{sent_by}</b>.
           text: Sent by %{sent_by}.
     kudos:
-      guest: A guest
-      guests: guests
+      guest: 
+        one: a guest
```

Comment:
This gives me a headache too:

https://github.com/otwcode/otwarchive/blob/master/app/mailers/kudo_mailer.rb#L35

        if !names.nil? && names.size > 0
          kudo_givers = names
          kudo_givers << guest_kudos(guest_count) unless guest_count == 0
        else
          kudo_givers << guest_kudos(guest_count).capitalize unless guest_count == 0
        end
        next if kudo_givers.empty?

See we have the capitalize so a guest  is correct as the A or the first letter of the list of users is capitalized.